go.mod

```diff
@@ -21,6 +21,7 @@ require (
 	github.com/imdario/mergo v0.3.15 // indirect
 	github.com/jbenet/go-context v0.0.0-20150711004518-d14ea06fba99 // indirect
 	github.com/kevinburke/ssh_config v1.2.0 // indirect
+	github.com/pelletier/go-toml v1.9.5 // indirect
```

Comment:
~ Looking at https://pkg.go.dev/github.com/pelletier/go-toml@v1.9.5#section-readme, it seems this version will be deprecated.  Something for us to be aware of as we add more dependencies.  (I also wonder if a regex search could work as well for this case to avoid adding it and maintaining any cves against it).

Reply:
Hmmmm I like that idea but also trying to generate the valid regex sounds like a nightmare 😆